lib/state/appState.js

```diff
@@ -89,6 +89,17 @@ appState.getInitial = function() {
   };
 };
 
+appState.hideUnavailableDevices = function() {
+  var currentOS = this.app.device._os; // "mac", "win", "android", "cros", "linux", or "openbsd"
```

Comment:
Yeah, sounds like more effort than it's worth. Maybe just add a comment so that other people reading the code from a fresh perspective (like me yesterday) have a hint as to where `this.app.device._os` is being set?